gateway/http.d/init.conf

```diff
@@ -3,6 +3,41 @@ init_by_lua_block {
     -- require('jit.p').start('vl')
     -- require('jit.dump').start('bsx', 'jit.log')
 
+    if os.getenv('CI') == 'true' then
+      pcall(require, 'luacov.runner')
+    end
+
+    local luacov = package.loaded['luacov.runner']
+
+    if luacov then
+        local pwd = os.getenv('PWD') .. package.config:sub(1, 1)
+        local pid = require("ngx.process").get_master_pid
+
+        if not pid then
+          do
+            local ffi = require("ffi")
+            ffi.cdef[[int getpid(void);]]
+            pid = ffi.C.getpid()
+          end
+        end
+
+        local config = { }
+        for _, option in ipairs({"statsfile", "reportfile"}) do
+          -- properly expand current working dir, workaround for https://github.com/openresty/resty-cli/issues/35
+          config[option] = pwd .. luacov.defaults[option]
+          luacov.defaults[option] = pwd
+        end
+
+        luacov.defaults.savestepsize = 3
+        jit.off()
```

Comment:
why?

Reply:
It was mentioned in several places like the mailing list and github. Looks like coverage can't really be tracked on the JITed code.